images/linux/scripts/SoftwareReport/SoftwareReport.Generator.ps1

```diff
@@ -68,6 +68,7 @@ $packageManagementList = @(
         (Get-NpmVersion),
         (Get-YarnVersion),
         (Get-PipxVersion),
+        (Get-ParcelVersion),
```

Comment:
[Parcel is a zero configuration build tool for the web](https://github.com/parcel-bundler/parcel), not the package management utility

Reply:
Thanks. I fixed it